chroot/run.go

```diff
@@ -1093,7 +1097,8 @@ func setupChrootBindMounts(spec *specs.Spec, bundlePath string) (undoBinds func(
 		}
 		subSys := filepath.Join(spec.Root.Path, m.Mountpoint)
 		if err := unix.Mount(m.Mountpoint, subSys, "bind", sysFlags, ""); err != nil {
-			return undoBinds, errors.Wrapf(err, "error bind mounting /sys from host into mount namespace")
+			logrus.Warningf("could not bind mount %q, skipping: %v", m.Mountpoint, err)
+			continue
```

Comment:
The commit log says some filesystems might not be accessible.  Is this a thing we're seeing now (with which filesystems?), or is it a future-proofing thing?

Reply:
I've seen the issue with `/sys/kernel/debug/tracing` on Fedora 32

Reply:
Fair enough.  I'd be strongly tempted to add `/sys/kernel` or some of its subdirectories to the set of paths that we mask out, too, but that would be a different PR.